courseraresearchexports/exports/api.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python
+
+# Copyright 2016 Coursera
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Coursera's wrapper for data exports API.
+"""
+
+import requests
+
+from courseraoauth2client import oauth2
+from courseraresearchexports.exports.constants import RESEARCH_EXPORTS_APP, \
+    RESEARCH_EXPORTS_API, CLICKSTREAM_API
+from courseraresearchexports.exports.utils import transform_response
```

Comment:
since this function also does error handling, our name should reflect that. 

maybe something like `requests_response_safe_transform`?